lib/datadog/appsec/contrib/rack/ext.rb

```diff
@@ -1,13 +1,26 @@
 # typed: ignore
+# frozen_string_literal: true
 
 module Datadog
   module AppSec
     module Contrib
       module Rack
         # Rack integration constants
         module Ext
-          APP = 'rack'.freeze
-          ENV_ENABLED = 'DD_TRACE_RACK_ENABLED'.freeze # TODO: DD_APPSEC?
+          APP = 'rack'
+          ENV_ENABLED = 'DD_TRACE_RACK_ENABLED' # TODO: DD_APPSEC?
+
+          RESPONSE_STATUS = 'response.status'
```

Comment:
These changes should be mirrored in `sig`

Reply:
Done